lib/jekyll/commands/new.rb

```diff
@@ -114,6 +116,30 @@ def site_template
         def scaffold_path
           "_posts/0000-00-00-welcome-to-jekyll.markdown.erb"
         end
+
+        def bundle_install(path)
+          Jekyll::External.require_with_graceful_fail "bundler"
+          $stdout.print "\nRun 'bundle install' with default theme? (Y/N)  "
```

Comment:
Jekyll logger inserts a newline at the end.. I proposed _print_ to have gets on same line